.github/workflows/ci.yaml

```
@@ -42,7 +42,7 @@ jobs:
   tests:
     strategy:
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11"]
+        python-version: ["3.10.14"]
```

This has come up before and I think it was decided to keep testing all versions.
However python 3.8 is going to EOL in 3 months so I think it's a relatively easy call to drop support for that.
https://endoflife.date/python

Even if product said we can drop support for everything but `3.10` I would still suggest we keep testing (at least unit-tests) on 3.11 and higher to keep our upgrade path clear.

Having said all that.
It would also be trivial to deal with the github API rate limit by making that test only run on one of python versions.

cc @Shinnnyshinshin @annabarr 
```suggestion
        python-version: ["3.9", "3.10.14", "3.11"]
```

---

@Kilo59 can you point me to where this was decided?

---

I don't understand why we would add unnecessary complexity for ourselves if we don't support running this outside of docker

---

@wookasz is it possible to know the version that people are running when downloading the `agent` from pypi?

---

@Shinnnyshinshin @wookasz 
https://pypistats.org/packages/great-expectations-cloud